manifests/base/cuda-rstudio-buildconfig.yaml

```diff
@@ -0,0 +1,156 @@
+    opendatahub.io/recommended-accelerators: '["nvidia.com/gpu"]'
+  name: cuda-rstudio-rhel9
+  labels:
+    opendatahub.io/dashboard: 'true'
+    opendatahub.io/notebook-image: 'true'
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+    - name: latest
+      annotations:
+        opendatahub.io/notebook-software: '[{"name":"CUDA","version":"11.8"},{"name":"Python","version":"v3.9"}]'
+        opendatahub.io/notebook-python-dependencies: '[{"name":"r-studio","version":"4.3"}]'
+      referencePolicy:
+        type: Source
+---
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true"
+  annotations:
+    opendatahub.io/notebook-image-url: "https://github.com/red-hat-data-services/notebooks/tree/main/rstudio"
+    opendatahub.io/notebook-image-name: "CUDA - R Studio"
+    opendatahub.io/notebook-image-desc: "R Studio workbench image, allows to integrated development environment for R, a programming language for statistical computing and graphics."
+    opendatahub.io/recommended-accelerators: '["nvidia.com/gpu"]'
+  name: cuda-rstudio-rhel9
+spec:
+  name: latest
+  lookupPolicy:
+    local: true
+---
+kind: BuildConfig
+apiVersion: build.openshift.io/v1
+metadata:
+  name: cuda-rhel9
+  labels:
+    app: buildchain
+    component: cuda-rhel9
+spec:
+  source:
+    type: Git
+    git:
+      uri: "https://github.com/red-hat-data-services/notebooks"
+      ref: main
+  strategy:
+    type: Docker
+    dockerStrategy:
+      dockerfilePath: "cuda/rhel9-python-3.9/Dockerfile"
+      noCache: true
+      from:
+        kind: "DockerImage"
+        name: "quay.io/modh/odh-base-rhel9:base-rhel9-python-3.9-20240131-378dafd"
+      env:
+      - name: USERNAME
+        valueFrom:
+          secretKeyRef:
+            name: rhel-subscription-secret
+            key: USERNAME
+      - name: PASSWORD
+        valueFrom:
+          secretKeyRef:
+            name: rhel-subscription-secret
+            key: PASSWORD
+      - name: SERVERURL
+        valueFrom:
+          secretKeyRef:
+            name: rhel-subscription-secret
+            key: SERVERURL
+      - name: BASEURL
+        valueFrom:
+          secretKeyRef:
+            name: rhel-subscription-secret
+            key: BASEURL
+  output:
+    to:
+      kind: ImageStreamTag
+      name: "cuda-rhel9:latest"
+  resources:
+    limits:
+      cpu: "1"
+      memory: 1Gi
+    requests:
+      cpu: "1"
+      memory: 1Gi
+  runPolicy: Serial
+  triggers:
+    - imageChange: {}
+      type: ImageChange
+---
+kind: BuildConfig
+apiVersion: build.openshift.io/v1
+metadata:
+  name: cuda-rstudio-rhel9
+  labels:
+    app: buildchain
+    component: cuda-rstudio-image
+spec:
+  source:
+    type: Git
+    git:
+      uri: "https://github.com/red-hat-data-services/notebooks"
+      ref: main
```

Comment:
Main? I'm thinking this should pull from the branch corresponding to the RHOAI version user has installed. In other words, the source branch needs to be a parameter.

Cc @jstourac, fyi, I think this is important to figure out.

Reply:
Hi Jiri, the main is the fallback branch, on the CF we will change to the release branch

Reply:
What's a CF?

Reply:
Code Freeze week